packages/common/src/activity-options.ts

```diff
@@ -106,6 +106,8 @@ export interface ActivityOptions {
    * When using the Worker Versioning feature, specifies whether this Activity should run on a
    * worker with a compatible Build Id or not. See {@link VersioningIntent}.
    *
+   * @defaultValue 'COMPATIBLE'
```

Comment:
I think it’s just `@default`

Reply:
tsdoc, right? i don't see @default, only: https://tsdoc.org/pages/tags/defaultvalue/

Reply:
Interesting... Yeah, it indeed look like `@defaultValue` is the right one. `@default` appears in [generated doc files](https://typescript.temporal.io/api/interfaces/client.BackoffOptions#factor), but that's because it is configured as a reserved discretionary block tag. The output (at least in VS Code) is different.

Reply:
@lorensr For this PR, please use `@default` anyway, as this is our current standard.

We can and should consider changing our standard, but that should be a repo-wide find and replace, and I would like to see the generated html doc before we commit to `@defaultValue`.

Reply:
`@default` is the jsdoc standard, tooling seems to support both. Haven’t checked how vs code displays them though